### PR TITLE
feat(jobs/pingcap-inc/enterprise-extension): add plugin job

### PR DESCRIPTION
**Pull request by wuhuizuo**
Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: pipelines/pingcap-inc/exterprise-extension/latest/pr-verify.groovy [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// REF: https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-pipeline
+// Keep small than 400 lines: https://issues.jenkins.io/browse/JENKINS-37984
+@Library('tipipeline') _
+
+final K8S_NAMESPACE = "jenkins-tidb"
+final GIT_CREDENTIALS_ID = 'github-sre-bot-ssh'
+final GIT_FULL_REPO_NAME = 'pingcap/tidb'
```
Comment (purelind) [inline pipelines/pingcap-inc/exterprise-extension/latest/pr-verify.groovy:7]: Correct  `'pingcap/tidb'` to `'pingcap-inc/exterprise-extension'`

File: pipelines/pingcap-inc/exterprise-extension/latest/pr-verify.groovy [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+final POD_TEMPLATE_FILE = 'pipelines/pingcap/tidb/latest/pod-pr-verify.yaml'
```
Comment (purelind) [inline pipelines/pingcap-inc/exterprise-extension/latest/pr-verify.groovy:8]: `'pipelines/pingcap/tidb/latest/pod-pr-verify.yaml'` --> `'pipelines/pingcap-inc/exterprise-extension/latest/pod-pr-verify.yaml'`

**Comment by wuhuizuo** [comment]
@purelind thanks, they are solved.

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**